core/arch/arm/plat-vexpress/platform_config.h

```diff
@@ -237,13 +242,19 @@
 #define DRAM0_TEERES_BASE	(DRAM0_BASE + DRAM0_SIZE)
 #define DRAM0_TEERES_SIZE	CFG_SHMEM_SIZE
 
-/* Secure RAM: SDP memory followed by OP-TEE secure memory */
-#define SECRAM_BASE		0x0e000000
-#define SECRAM_SIZE		0x01000000
+/*
+ * Secure RAM:     SDP disabled    |  SDP enabled
+ *   ARM-TF        1M@0x0e000000   |   1M@9x0e000000
+ *   SDP memory        n.a         |   2M@0x0e100000
```

Comment:
Wouldn't it be easier to have SDP memory after TA RAM? That way it's only the TA RAM size that changes.